pkg/berglas/create.go

```diff
@@ -77,6 +78,19 @@ func (c *Client) Create(ctx context.Context, i *CreateRequest) (*Secret, error)
 		return nil, errors.New("missing plaintext")
 	}
 
+	_, err := c.storageClient.
```

Comment:
I'm really hesitant about updating this API contract. People may be depending on the fact that create overwrites. What do you think?

Reply:
Hmm, imho, I don't like create overwriting without explicit request to do so. Now that update exists, in most (all?) cases you'd use that over create when updating a secret. However, yes, it is a breaking change. Can we hear from other users? A possible solution is to add a `overwrite` flag (default `true`) to the request, but let CLI module pass in `false` still.

Reply:
The challenge becomes automation or a situation where you need to assert a secret exists. If we change the semantics of create to error if a secret already exists and keep the semantics of update erroring if the secret does not exist, then the user has to introduce a read lookup and then branching logic in their application.

Reply:
I think branching logic is the right way. If create is used in automation and intends to overwrite then a CLI flag `--overwrite` should work.. 

There are 3 implications of `create`:
1. Creates a new secret
1. Updates an existing secret after the value is read and modified appropriately
1. Unintentionally clobbers an existing object or secret

`#1` is still fine - that's the correct way to create a new secret.
`#2` should move to using `edit` - breaking change, however a `--overwrite` flag would help. Alternatively, a `--no-overwrite` flag which should in the future become default, thus preserving the contract for `create` for now.
`#3` is just an error which wasn't surfaced until now.

Reply:
Let's add a `--create-if-missing` flag to update (and an associated field on the struct for the API).

Reply:
Sounds good.